Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/service/PersistedTasksService.java

```diff
@@ -51,20 +56,15 @@ public PersistedTasksService(
     }
 
     @Override
-    public Observable<Event<Tasks>> getTasksEvents() {
+    public Observable<Event<Tasks>> getTasksEvent() {
```

Comment:
I kind of don't like `getTasksEvents` because we might not actually get any events from the observable. How about `createTasksEventObservable` or `observableOfTasksEvent` ?